src/scale.ts

```diff
@@ -458,7 +459,7 @@ export interface SchemeParams {
    *
    * For the full list of supported schemes, please refer to the [Vega Scheme](https://vega.github.io/vega/docs/schemes/#reference) reference.
    */
-  name: string | SignalRef;
+  name: string | ColorScheme | SignalRef;
```

Comment:
This is great. Should we even allow string?

Reply:
Good point! I went with a more permissive type because I wasn't sure what the down stream deps were from this type, but if this seems fine to you then a more assertive seemed great!